lib/smart_app_launch/smart_access_brands_group.rb

```diff
@@ -0,0 +1,23 @@
+require_relative 'smart_access_brands_validation_group'
+require_relative 'smart_access_brands_retrieval_group'
+
+module SMARTAppLaunch
+  class SMARTAccessBrandsGroup < Inferno::TestGroup
+    id :retrieve_and_validate_smart_access_brands
+    title 'Retrieve and Validate SMART Access Brands Bundle'
+    description %(
+      Verify that the Brand Bundle Publisher makes its User-access Brands publication publicly available
+      in the format defined by the [User Access Brand Bundle Profile](https://build.fhir.org/ig/HL7/smart-app-launch/StructureDefinition-user-access-brands-bundle.html)
```

Comment:
Also references build.fhir.org.  Get stable links for all of these.  Because we are specifically pointing to 2.2, use https://hl7.org/fhir/smart-app-launch/STU2.2/ as the base.  I won't comment on all of these... just do a search and fix all of them :)